01week/datatypes.js

```diff
@@ -0,0 +1,98 @@
+// Write a JavaScript program to display the current day and time.
+
+const currentDayandTime =()=> {
+
+const currentDate = new Date();
+const day = currentDate.getDate();
+const month = currentDate.getMonth();
+const year = currentDate.getFullYear();
+
+const hours = currentDate.getHours();
+const minutes = currentDate.getMinutes();
+const seconds = currentDate.getSeconds();
+
+const time = currentDate.getUTCDate();
+
+return(month + "/" + day + "/" + year + "   " + "Time:" + hours +":" + minutes +":" + seconds );
+}
+
+console.log(currentDayandTime());
+
+
+// Write a JavaScript program to convert a number to a string.
+
+const numbertoString = (num1) => {
+  const s = num1.toString();
+
+  return(s);
+}
+
+console.log(numbertoString(15));
+
+// Write a JavaScript program to convert a string to the number.
+
+const stringtoNumber = (s1) => {
+  const s2 = parseInt(s1);
+
+  return(s2);
+}
+
+console.log(stringtoNumber('44.hello'));
+
+/*Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+Boolean
+Null
+Undefined
+Number
+NaN
+String
+*/
+
+const dateTypes =() => { 
+  const s = 'thisisstring';
+  const b = true;
+  const n = 15;
+  const noValue = null;
+  return(typeof b); 
```

Comment:
This function evaluates the data types of the variables s,b, n, and noValue. I need it to take in a parameter and evaluate it's data type